suite-native/atoms/src/Input/SearchInput.tsx

```diff
@@ -54,7 +46,13 @@ const inputWrapperStyle = prepareNativeStyle<InputStyleProps>((utils, { isFocuse
     ],
 }));
 
-export const SearchInput = ({ value, onChange, placeholder, isDisabled = false }: InputProps) => {
+export const SearchInput = ({
+    value,
+    onChange,
+    placeholder,
+    isDisabled = false,
```

Comment:
default values props should always be as last prop item

Reply:
Order changed in a14e10331.